test/CSAccounting.t.sol

```diff
@@ -536,7 +536,7 @@ abstract contract CSAccountingBondStateBaseTest is
         });
     }
 
-    uint256[] public defaultCurve = [2 ether, 3 ether];
+    uint256[] public curveWithDiscount = [2 ether, 3 ether];
```

Comment:
But why?

Reply:
because it's not a default curve
default curve is what was set in the initializer (2 ether per key in these tests)